src/pages/api/fareType.ts

```diff
@@ -11,15 +11,23 @@ export default (req: NextApiRequest, res: NextApiResponse): void => {
         }
 
         if (req.body.fareType) {
+            const cookieValue = JSON.stringify({
+                errorMessage: '',
+                uuid: getUuidFromCookie(req, res),
```

Comment:
Do we need to add an errorMessage and uuid here?

Reply:
yes ideally to reset it if required. And the uuid is used to match the if cookies match if required. This keeps it consistent with all cookies

Reply:
Cool!